packages/kbn-unified-data-table/src/utils/get_render_cell_value.tsx

```diff
@@ -47,10 +47,7 @@ export const getRenderCellValueFn = ({
   externalCustomRenderers?: CustomCellRenderer;
   isPlainRecord?: boolean;
 }) => {
-  /**
-   * memo is imperative here otherwise the cell will re-render on every hover on every cell
-   */
-  return memo(function UnifiedDataTableRenderCellValue({
+  return function UnifiedDataTableRenderCellValue({
```

Comment:
Had to remove `memo` here because EUI fails the unit tests when  a memoized components is passed to the Data Grid with error that `typeof renderCellValue !== 'function'`

Reply:
Just curious, is the named function here just to help with debugging?

Reply:
yes exactly. Otherwise, it appears as `Anonymous` in React Dev tools. Not other reason.